pkg/postage/service.go

```diff
@@ -44,58 +43,36 @@ type Service interface {
 // stores the active batches.
 type service struct {
 	lock         sync.Mutex
-	store        storage.StateStorer
+	store        storage.Store
 	postageStore Storer
 	chainID      int64
-	issuers      []*StampIssuer
+	issuersInUse map[string]any
 }
 
 // NewService constructs a new Service.
-func NewService(store storage.StateStorer, postageStore Storer, chainID int64) (Service, error) {
-	s := &service{
+func NewService(store storage.Store, postageStore Storer, chainID int64) Service {
+	return &service{
 		store:        store,
 		postageStore: postageStore,
 		chainID:      chainID,
+		issuersInUse: make(map[string]any),
```

Comment:
what is the purpose of the map? I think we can get rid of it.

Reply:
We use it to track which issuers are being used